app/controllers/api/v1/customers_controller.rb

```diff
@@ -88,17 +88,20 @@ def customer_params
         attributes = params.require(:customer).permit(
           :email, :enterprise_id,
           :code, :first_name, :last_name,
-          :billing_address, shipping_address: [
+          :billing_address,
+          shipping_address: [
             :phone, :latitude, :longitude,
             :first_name, :last_name,
             :street_address_1, :street_address_2,
             :postal_code, :locality,
             { region: [:name, :code], country: [:name, :code] },
-          ]
+          ],
+          tags: [],
```

Comment:
This parameter permission was missing, the tags were passed from the root parameters to the nested parameters [here](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#create-a-pull-request).
Maybe, we will want to pass the `tag_list` parameter directly